lib/Reader/readerir.cpp

```diff
@@ -575,6 +576,98 @@ void GenIR::callMonitorHelper(bool IsEnter) {
                  MethodSyncHandle, (IRNode *)SyncFlag);
 }
 
+void GenIR::insertIRForUnmanagedCallFrame() {
+  // There's nothing more to do if we've already inserted the necessary IR.
+  if (Reader.unmanagedCallFrame != nullptr) {
```

Comment:
Should it be Reader.UnmanagedCallFrame (with a capital U)? How does it compile with this typo?